TypeCobol.LanguageServer/TypeCobolServer.cs

```diff
@@ -256,17 +256,22 @@ protected override void OnShutdown()
             base.OnShutdown();
         }
 
+        protected virtual void OnDidChangeConfiguration(string[] arguments)
+        {
+            this.Workspace.DidChangeConfigurationParams(arguments);
+        }
         protected override void OnDidChangeConfiguration(DidChangeConfigurationParams parameters)
         {
             if (parameters.settings is Newtonsoft.Json.Linq.JArray)
             {
                 Newtonsoft.Json.Linq.JArray array = parameters.settings as Newtonsoft.Json.Linq.JArray;
-                IEnumerable<string> arguments = array.Select(t => t.ToString());
-                this.Workspace.DidChangeConfigurationParams(arguments);
+                IEnumerable<string> argsEnum = array.Select(t => t.ToString());
+                string[] arguments = argsEnum.ToArray<string>();
+                OnDidChangeConfiguration(arguments);
             }
             else
             {
-                this.Workspace.DidChangeConfigurationParams(parameters.settings.ToString());
+                OnDidChangeConfiguration(parameters.settings.ToString().Split(' '));                
```

Comment:
Unwanted whitespace at the end of the line

Reply:
Done in d7cc3ff